frontend/src/pages/EmployeeProfile/styled.ts

```diff
@@ -109,16 +115,18 @@ export const UserInfoText = styled('span')({
   },
 });
 
+export const EmployeeInfoItem = styled(ListItem)({
+  alignItems: 'baseline !important',
```

Comment:
Do we really need to add an important here?

Reply:
@k-krasilnikova yep, unfortunately it doesn't work correctly without this declaration, I also tried to use global MUI classes to avoid using `!important`, but still